Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -329,6 +329,13 @@ void GcodeSuite::G28(const bool always_home_all) {
         delayed_move_time = 0;
         active_extruder_parked = true;
 
+        if (dxc_is_duplicating()) {
+          extruder_duplication_enabled = IDEX_saved_duplication_state;
+          extruder_duplication_enabled = false;
```

Comment:
Are you as curious as I am as to why this is setting the same variable twice with two different values? It was in the previous code also.

Reply:
Changes made in 7680b64aa6 and 0780913848